docker/lookaside.go

```diff
@@ -178,8 +185,16 @@ func (config *registryConfiguration) signatureTopLevel(ref dockerReference, writ
 			return url
 		}
 	}
-	logrus.Debugf(" No signature storage configuration found for %s", ref.PolicyConfigurationIdentity())
-	return ""
+	if euid := rootless.GetRootlessEUID(); euid != 0 {
```

Comment:
> `// config.signatureTopLevel returns an URL string configured in config for ref, for write access if “write”.`

i.e. this could return `nil` if `config` does not specify a configuration.

Moving these defaults up to `configuredSignatureStorageBase` would also avoid the unnecessary roundtrip between `url.URL`, a `string`, and back.

--- 

Either way the documentation of the function needs updating, if only to update the “if no signature storage should be used.” part.